jcommune-view/jcommune-web-controller/src/test/java/org/jtalks/jcommune/web/controller/PrivateMessageControllerTest.java

```diff
@@ -78,6 +81,7 @@
     private static final JCUser JC_USER = new JCUser(USERNAME, "123@123.ru", "123");
     private static final String TITLE = "test title";
     private static final String BODY = "test body";
```

Comment:
You can refactor the tests to get rid of these stupid constants here.. But it's up to you.